recipes/r-climate4r.datasets/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set version = '0.0.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-climate4r.datasets
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url: https://github.com/SantanderMetGroup/climate4R.datasets/archive/refs/tags/v0.0.1.tar.gz
+  sha256: 75fea46d84e61a45d692a69a2fd11db0cf9849197905d50df079890a0824124c
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}git
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base >=3.5,<4
```

Comment:
This results into R 3.6 builds being done, which is not really supported by conda-forge anymore. Is that intended? Can be lifted also after the merge to get supported R 4 and R 4.1 builds.